flink-formats/flink-avro/pom.xml

```diff
@@ -185,17 +185,6 @@ under the License.
 						<configuration combine.self="override">
 							<dependencyReducedPomLocation>${project.basedir}/target/dependency-reduced-pom.xml</dependencyReducedPomLocation>
 							<promoteTransitiveDependencies>true</promoteTransitiveDependencies>
```

Comment:
Btw, is this `promote` needed here?

Reply:
i don't really know, but it has been there for a while afaik.

Reply:
We can probably remove the complete shading entry in that pom. Seem not to do anything anyways. Promoting transitive dependencies when nothing except `force-shading` is shaded into the jar should be a no-op. (theoretically, one never knows with the shade plugin ;-) )